### PR TITLE
Display parameter description in prompt mode

### PR DESCRIPTION
**Pull request by ernilambar**
See https://github.com/wp-cli/wp-cli/issues/3121

### CODE REVIEW

File: php/WP_CLI/Dispatcher/Subcommand.php [hunk 1]
```diff
@@ -148,6 +148,10 @@ private function prompt_args( $args, $assoc_args ) {
 		foreach( $spec as $key => $spec_arg ) {
 
 			$current_prompt = ( $key + 1 ) . '/' . count( $spec ) . ' ';
+			$description_match = preg_match( '/' . preg_quote( $spec_arg['token'] ) . '\r?\n\:\s(.*)/', $this->longdesc, $matches );
```
Comment (danielbachhuber) [inline php/WP_CLI/Dispatcher/Subcommand.php]: You should use `WP_CLI\DocParser` here.

File: php/WP_CLI/Dispatcher/Subcommand.php [hunk 2]
```diff
@@ -148,6 +148,10 @@ private function prompt_args( $args, $assoc_args ) {
...
+			if ( $description_match ) {
+				$current_prompt = trim( $matches[1] ) . "\n" . $current_prompt;
```
Comment (danielbachhuber) [inline php/WP_CLI/Dispatcher/Subcommand.php]: `"\n` should be `PHP_EOL`

**Comment by danielbachhuber** [comment]
In addition to the two comments I left, can you share an example for how this renders?

File: php/WP_CLI/Dispatcher/Subcommand.php (hunk 1 above)
Comment (ernilambar) [inline php/WP_CLI/Dispatcher/Subcommand.php]: `$this->docparser->get_arg_desc( $spec_arg['name'] );` should return argument description, no?

Comment (danielbachhuber) [inline php/WP_CLI/Dispatcher/Subcommand.php]: > should return argument description, no?

Yes, it should.

**Comment by ernilambar** [comment]
I am closing this PR for now. May be this is above my brain. If I am able to resolve it then I will submit new PR.

**Comment by danielbachhuber** [comment]
Sounds good!

**Pull request closed without merging**